static/js/shared/ga_events.ts

```diff
@@ -97,6 +98,22 @@ export const GA_EVENT_TOOL_CHART_OPTION_CLICK = "tool_chart_option_click";
  */
 export const GA_EVENT_NL_SEARCH = "explore_search_q";
 
+/**
+ * Triggered when detection results are returned in NL search.
+ * Parameters:
+ *   "query": search query
+ *   "time_ms": e2e request/response timing in ms.
+ */
+export const GA_EVENT_NL_DETECT_FULFILL = "explore_detect_fulfill";
+
+/**
+ * Triggered when detection results are returned in NL search.
+ * Parameters:
+ *   "topic": array of topics
+ *   "time_ms": e2e request/response timing in ms.
+ */
+export const GA_EVENT_NL_FULFILL = "explore_fulfill";
```

Comment:
fulfill is based on topic + place, should we also include places as a parameter?